src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -217,7 +217,7 @@ public override bool CanWriteToken
         /// <summary>
         /// Gets or sets the token lifetime in minutes.
         /// </summary>
-        /// <remarks>Used by <see cref="CreateToken(string, string, ClaimsIdentity, DateTime?, DateTime?, SigningCredentials)"/> to set the default expiration ('exp'). <see cref="DefaultTokenLifetimeInMinutes"/> for the default.</remarks>
+        /// <remarks>Used by <see cref="CreateJwtSecurityTokenPrivate(string, string, ClaimsIdentity, DateTime?, DateTime?, DateTime?, SigningCredentials)"/> to set the default expiration ('exp'). <see cref="DefaultTokenLifetimeInMinutes"/> for the default.</remarks>
```

Comment:
Are we referencing a private member here? We should only be referencing public types and members.